mentoring/programs/mentoring-lead.md

```diff
@@ -0,0 +1,58 @@
+
+## Responsibilities
+ The role lead should:
+
+- Keep a calendar of dates for each program (when to submit our project name, when to submit the project scope, when selecting period is, etc) throughout the   year and
+- Coordinate all submissions and participation in each one
+- Go to sig meetings and slack channels to proactively help with scoping and planning work / job descriptions
+- Help onboard interns to the k8s specific community items like coordinate a hello for them at the k8s monthly community meeting
+- Help off board interns to the k8s specific community items like demo at monthly community meeting
+
+
+### Role Description
+
+Major responsibilities includes a commitment to: 
+
+- Be proactive in information sharing, through the mailing list/slack channels, regarding various mentorship cohorts
+- Be a bridge between the CNCF staff, mentors, and mentees from kubernetes project
+- Be an active contributor, who could identify parts of the project that could get help from mentorship programs
+- Be a go-to person for all SIGs to help them with their mentorship proposals
+- Be welcoming of everyone by being your unique self
+- Be inclusive on contributors, especially with new contributors joining kubernetes project as part of the mentoring process
+- Be a leader of the mentoring sub-project with a commitment to its charter
+
+### Minimum Skills and Requirements
+
+Skills:
+
+- Be comfortable with general GitHub workflows. You'll be working inside of multiple repositories with different workflows and helping others on the team with troubleshooting.
+- Be a team player, making the project welcoming for new mentees and helping other members of the project interested to part in mentorship with the process.
+- Be willing and capable of saying "no" as necessary, but try to get to a "yes, and" to help the community achieve its goals. In short, be empathetic.
+- Strong written and verbal communication skills. A willingness to meet (a lot) of new people is important for this work.
+
+Requirements:
+
+- Member of the [Kubernetes GitHub Org]
+- [Reviewer] in the mentoring folder
+- Not Necessary, but a previous mentee/mentor in any of the mentorship programme is preferrable
+
+#### Expected Time Investment
+
+- Four hours a week as a minimal target, might be two hours more during the proposal submission perios
+
+#### Duration  
+
+Ideally, no lead should be in the same position indefinitely. With that in mind,the project would like to see new leadership every 24 months to keep a fresh perspective. 
```

Comment:
We should also add a point in the list of responsibilities for the lead to nominate a new lead within 24 months, so this is codified in policy.